platform/viewer/src/routes/Mode/Mode.jsx

```diff
@@ -15,7 +15,9 @@ export default function ModeRoute({
   dataSourceName,
   extensionManager,
   servicesManager,
+  hotkeysManager
 }) {
+  const isHotkeysLoaded = useHotkeys(hotkeysManager, mode.hotkeys);
```

Comment:
Easier to just drop in `useEffect` that inits/clears? Return value probably not needed

Reply:
Totally fine removing the custom hook and using it directly =)